buildSrc/src/main/kotlin/com/bugsnag/android/BugsnagBuildPlugin.kt

```diff
@@ -115,6 +115,7 @@ class BugsnagBuildPlugin : Plugin<Project> {
             isCheckAllWarnings = true
             baseline(File(project.projectDir, "lint-baseline.xml"))
             disable("GradleDependency", "NewerVersionAvailable")
+            ignore("InvalidPackage")
```

Comment:
dsl-json is designed to run on many systems, and doesn't require anything outside of Android OS standard library. But it will query for extra things that might be present, which is why it's showing up in the warnings.

Reply:
This needs addressing by adding to the baseline file - the warning cannot be suppressed like this as it runs the risk of missing similar issues when adding new dependencies.